ec/acc/acc_prereq.go

```diff
@@ -118,6 +119,7 @@ func newAPI() (*api.API, error) {
 		Host:          host,
 		SkipTLSVerify: insecure,
 		Retries:       ec.DefaultHTTPRetries,
+		Timeout:       5 * time.Minute,
```

Comment:
This won't have the desired effect. The timeout here controls the http client timeout before it the Go `http.Client` instance cancels an individual the request.

To make matters worse, there's no timeout parameter in the shutdown operation so we can't work around it on the client side: https://github.com/elastic/cloud-sdk-go/blob/master/api/apidocs-user.json#L1731-L1756